### PR TITLE
Peer review

### PR DESCRIPTION
**Pull request by bnjn**
Adding normalize.css, blank index.html and stylesheet.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    <section id='project-section' class='projects'>
+      <h2>Projects</h2>
+      <div class='project-container'>
+        <div>
+          <div class='button-overlay'>
+            <button type="button" name="button" class='about-button'>About</button>
+            <button type="button" name="button" onclick="window.open('https://github.com', '_blank');">GitHub</button>
+          </div>
+          <img src='img/placeholder.png' alt='cat'>
+        </div>
```
Comment (maxdyer97) [inline index.html:41]: Could be more indented to the left to join up with the div element above it perhaps.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      <h2>About</h2>
+      <div class='about-flexbox'>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed scelerisque nulla et velit elementum, ut pretium enim pellentesque. Ut pretium venenatis tempus. Integer varius tempus feugiat. Vivamus a imperdiet risus. Fusce eu tincidunt purus. Ut bibendum sapien eros, ut viverra risus semper ac. Fusce fringilla metus at quam dignissim, a fermentum lacus lacinia. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer elementum lorem vel sapien iaculis eleifend. Nunc ullamcorper facilisis risus ac luctus. Nam finibus turpis nibh, vel interdum arcu lacinia vitae. Sed non nunc sapien. Donec cursus orci ligula, nec porttitor mauris feugiat eu. Aenean tellus lorem, iaculis in sodales quis, tincidunt lobortis purus. Sed non odio quis nunc rutrum pellentesque vitae euismod metus. Sed aliquam mauris sit amet ex aliquet porttitor.</p>
+        <img src='img/profile-photo.png' alt='photo of ben johnson'>
+      </div>
+    </section>
+    <section id='project-section' class='projects'>
+      <h2>Projects</h2>
+      <div class='project-container'>
+        <div>
```
Comment (maxdyer97) [inline index.html:35]: Not sure if this div has a closing tag or if it's accidental.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+#upbutton {
+  display: none; /* Hidden by default */
+  position: fixed; /* Fixed/sticky position */
+  bottom: 20px; /* Place the button at the bottom of the page */
+  right: 30px; /* Place the button 30px from the right */
+  z-index: 99; /* Make sure it does not overlap */
+  outline: none; /* Remove outline */
+  padding: 15px; /* Some padding */
```
Comment (maxdyer97) [inline css/style.css:205]: Really like the commenting here as it's a lot of CSS for the top button function you have.

**Review by maxdyer97 — APPROVED** [review]
Other than my couple of comments about div stuff, looks spot on!

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
+* {
+  box-sizing: border-box;
+}
+
+h1, h2, p, body, img, div {
```
Comment (maxdyer97) [inline css/style.css:13]: Love this use of DRY, gathering multiple elements in one { }

**Review by maxdyer97 — COMMENTED** [review]
Literally nothing bad to comment on about from confusing divs. (I could be mis-reading it though)

**Comment by maxdyer97** [comment]
Overall website is absolutely ace, big fan of the blinking _ after your name. In particular, the JS function to go to the top is such a QoL improvement, scrolling all the time isn't ideal so love this. Keep at it!

File: index.html (hunk 1 above)
Comment (bnjn) [reply index.html:41]: Resolved in person.

File: css/style.css (hunk 4 above)
Comment (bnjn) [reply css/style.css:13]: Thanks!

**Pull request merged**